### PR TITLE
refine gcd & lcm docs

### PR DESCRIPTION
**Pull request by luotao1**
### PR types

Others
### PR changes

Docs
### Describe

fix https://github.com/PaddlePaddle/Paddle/pull/37819#pullrequestreview-828392239
- gcd
image

- lcm
image

### CODE REVIEW

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -2797,7 +2797,9 @@ def gcd(x, y, name=None):
         gcd(0,0)=0, gcd(0, y)=|y|
 
     Args:
-        x, y (Tensor): An N-D Tensor, the data type is int8，int16，int32，int64，uint8. 
+        x (Tensor): An N-D Tensor, the data type is int8，int16，int32，int64，uint8. 
+            If x.shape != y.shape, they must be broadcastable to a common shape (which becomes the shape of the output).
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: 这一句也以note的格式给出来吧，就不用写两遍了，下面同理

Comment (luotao1) [reply python/paddle/tensor/math.py]: done

Comment (luotao1) [reply python/paddle/tensor/math.py]: done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**